### PR TITLE
#1666 added ability to deny resume

### PR DESCRIPTION
**Pull request by krzyk**
#1666 
* added `deny @login` option
* enabled test
* added stakeholder for denying

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1699](https://github.com/zerocracy/farm/issues/1699) is now in scope, role is `REV`

File: src/main/groovy/com/zerocracy/stk/pmo/profile/deny_resume.groovy [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  if (resumes.examiner(login) != author) {
+    throw new SoftException(
+      new Par('You are not the examiner of %s, see §1').say(login)
+    )
+  }
+  User.Smart user = new User.Smart(
+    new ExtGithub(farm).value().users().get(login)
+  )
+  if (!user.exists()) {
+    throw new SoftException(
```
Comment (g4s8) [inline src/main/groovy/com/zerocracy/stk/pmo/profile/deny_resume.groovy]: @krzyk I think we need to remove the resume in such case

File: src/main/groovy/com/zerocracy/stk/pmo/profile/deny_resume.groovy [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+      new Par(
+        'We can\'t find @%s in Github: https://github.com/%1$s: %s'
+      ).say(login)
+    )
+  }
+  if (user.type() != 'User') {
+    throw new SoftException(
```
Comment (g4s8) [inline src/main/groovy/com/zerocracy/stk/pmo/profile/deny_resume.groovy]: @krzyk same here (remove resume)

**Comment by krzyk** [comment]
@g4s8 updated

**Comment by krzyk** [comment]
@g4s8 ping

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#1699` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#1699](https://github.com/zerocracy/farm/issues/1699) is now out of scope

**Pull request merged**